apps/ledger-live-mobile/src/newArch/features/WalletSync/components/Activation/index.tsx

```diff
@@ -5,19 +5,25 @@ import { Flex, Text } from "@ledgerhq/native-ui";
 import { useTranslation } from "react-i18next";
 import { useTheme } from "styled-components/native";
 import { useInitMemberCredentials } from "../../hooks/useInitMemberCredentials";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import { Linking } from "react-native";
 
 type Props = { onSyncMethodPress: () => void };
 
 const Activation: React.FC<Props> = ({ onSyncMethodPress }) => {
   const { colors } = useTheme();
   const { t } = useTranslation();
+  const walletSyncFF = useFeature("llmWalletSync");
+  const learnMoreLink = walletSyncFF?.params?.learnMoreLink;
 
   useInitMemberCredentials();
 
   const onPressSyncAccounts = () => onSyncMethodPress();
 
   const onPressHasAlreadyCreatedAKey = () => onSyncMethodPress();
 
+  const onPressLearnMore = () => learnMoreLink && Linking.openURL(learnMoreLink);
```

Comment:
you may have an eslint problem here for `no-unused-expressions` rule. For a better readability and code maintainability the better is to use an if bloc and invoke your openURL function there.